aspnetcore/client-side/libman/libman-vs.md

```diff
@@ -0,0 +1,201 @@
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 08/13/2018
+uid: client-side/libman/libman-vs
+---
+# Use LibMan with ASP.NET Core in Visual Studio
+
+By [Scott Addie](https://twitter.com/Scott_Addie)
+
+Visual Studio has built-in support for LibMan in ASP.NET Core projects, including:
+
+* Support for configuring and running LibMan restore operations on build
+* Menu items for triggering LibMan restore and clean operations
+* Search dialog for finding libraries and adding the files to your project
+* Editing support for *libman.json*&mdash;the LibMan manifest file
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/client-side/libman/samples/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Prerequisites
+
+* Visual Studio 2017 version 15.8 or later with the **ASP.NET and web development** workload
+
+## Add library files
+
+Library files can be added to an ASP.NET Core project in two different ways:
+
+1. [Use the Add Client-Side Library dialog](#use-the-add-client-side-library-dialog)
+1. [Edit the LibMan manifest](#edit-the-libman-manifest)
+
+### Use the Add Client-Side Library dialog
+
+Follow these steps to install a client-side library:
+
+* In **Solution Explorer**, right-click the project folder in which the file(s) should be added. Choose **Add** > **Client-Side Library**. The **Add Client-Side Library** dialog appears:
+
+  ![Add Client-Side Library dialog](_static/add-library-dialog.png)
+
+* Select the library provider from the **Provider** drop down. CDNJS is the default provider.
+* Type the library name to fetch in the **Library** text box. IntelliSense provides a list of libraries beginning with the provided text.
+* Select the library from the IntelliSense list. Notice the library name is suffixed with the `@` symbol and the latest stable version known to the selected provider.
+* Decide which files to include:
+  * Select the **Include all library files** radio button to include all of the library's files.
+  * Select the **Choose specific files** radio button to include a subset of the library's files. When the radio button is selected, the file selector tree becomes usable. Check the boxes to the left of the file names to download.
+* Specify the project folder, for storing the files, in the **Target Location** text box. As a recommendation, store each library in a separate folder.
+
+  The suggested **Target Location** folder is based on the location from which the dialog launched:
+
+  * If launched from the project root:
+    * *wwwroot/lib* is used if *wwwroot* exists.
+    * *lib* is used if *wwwroot* doesn't exist.
+  * If launched from a project folder, the corresponding folder name is used.
+
+  The folder suggestion is suffixed with the library name. The following table illustrates folder suggestions when installing jQuery in a Razor Pages project.
+  
+  |Launch location |Suggested folder  |
```

Comment:
Table is a bit messy.

Reply:
Any suggestions to improve that?

Reply:
I just meant the layout ... the spacing. I'm a neat-freak on tables. :smile: I align the columns (unless the cell content is wide) and make the spacing around pipes consistent.

Reply:
I'll borrow that idea from you. I suppose that makes maintenance easier down the road.